### DISCUSSION TITLE
Snippets Manager (File Based)

### DISCUSSION THREAD

**Discussion by blakecannell** [johnlindquist/kit#1240] [Share]
Simple file based snippets manager:

A few things to note:
- File paths are currently specific to Windows. I will update for other envionments.
- This assumes a `snippets` folder exists within your home folder. Is there any way to make this configurable (within the manager itself)? This is of course quite simple to set as a constant at the top of the file if not.

https://gist.github.com/blakecannell/bea79f6c69103a410181802855855aa4

**Comment by blakecannell** [comment]
Also, something small I've noticed. When running `edit`, it will open in VSCode (my default), along with an additional empty file. I can create an issue for this if need be.

edit: Existing issue: https://github.com/johnlindquist/kit/issues/1211

**Comment by johnlindquist** [comment]
@blakecannell The journal script has an example of prompting the user for a path then saving it to the .env:

https://github.com/johnlindquist/kit-examples/blob/main/scripts/journal.js

**Reply by blakecannell** [reply]
Thanks @johnlindquist I'll look in to it!
